src/foundation/Icon/Generated/IconHighlight.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { memo } from "react";
+import { GeneratedIconProps } from "@foundation/Icon/IconProps";
+import { IconSize, IconSizeMap } from "@foundation/Icon/IconSize";
+
+function IconHighlight(props: GeneratedIconProps): React.ReactElement<GeneratedIconProps> {
+    const customClassName = [
+        "tw-flex tw-items-center tw-justify-center tw-fill-current",
+        IconSizeMap[props.size || IconSize.Size16],
+    ].join(" ");
+    return (
+        <svg
+            viewBox="0 0 19 19"
+            fill="none"
```

Comment:
```suggestion
            fill="currentColor"
```

Reply:
You can fix the issues Nadia mentionned in the svg then run `npm run build`

Reply:
done!